queries/quote.go

```diff
@@ -27,15 +26,13 @@ func scanRows(rows *sql.Rows) ([]*models.Quote, error) {
 
 func InsertQuote(quote models.Quote) error {
 	statement, err := db.DB.Prepare("INSERT INTO quote(quote, character, season, episode, title) VALUES($1, $2, $3, $4, $5)")
-
 	if err != nil {
-		fmt.Print(err)
+		slog.Error(err)
```

Comment:
can we make this error log more verbose? maybe something like this

```suggestion
		slog.Error("InsertQuote", "prepare SQL statement", err)
```